ui/desktop/electron-app/src/ipc/handlers.js

```diff
@@ -119,7 +119,7 @@ ipcMain.on('createTerminal', (event, payload) => {
   const { id, cols, rows } = payload;
   const { sender } = event;
   const terminalShell =
-    os.platform() === 'win32' ? 'powershell.exe' : '/bin/bash';
+    os.platform() === 'win32' ? 'powershell.exe' : process.env.SHELL;
```

Comment:
I think we should still default to bash just in case, even though any common unix system will have the variable defined. But we can't assume it always will be for some custom flavor of unix or if the user modified it for whatever reason. 
```suggestion
    os.platform() === 'win32' ? 'powershell.exe' : process.env.SHELL || '/bin/bash';
```

Reply:
Second this change.